processing/src/main/java/io/druid/segment/incremental/OnheapIncrementalIndex.java

```diff
@@ -127,45 +130,75 @@ protected Integer addToFacts(
       ThreadLocal<InputRow> in
   ) throws IndexSizeExceededException
   {
-    Integer rowOffset;
-    synchronized (this) {
-      rowOffset = numEntries.get();
-      if(rowOffset >= maxRowCount && !facts.containsKey(key)) {
+    final Integer priorIdex = facts.get(key);
```

Comment:
crap, missed that one, I thought I got them all, will change shortly.

On Fri, Jan 16, 2015 at 11:39 AM, Fangjin Yang <notifications@github.com>
wrote:

> In
> processing/src/main/java/io/druid/segment/incremental/OnheapIncrementalIndex.java
> <https://github.com/druid-io/druid/pull/1027#discussion-diff-23103861>:
>
> > @@ -127,45 +130,75 @@ protected Integer addToFacts(
> >        ThreadLocal<InputRow> in
> >    ) throws IndexSizeExceededException
> >    {
> > -    Integer rowOffset;
> > -    synchronized (this) {
> > -      rowOffset = numEntries.get();
> > -      if(rowOffset >= maxRowCount && !facts.containsKey(key)) {
> > +    final Integer priorIdex = facts.get(key);
>
> sorry this is my OCD coming out, but can we call this Index not Idex?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/druid-io/druid/pull/1027/files#r23103861>.
>



-- 

<http://www.metamarkets.com/> Charles Allen PhDSr. Software Engineer|
METAMARKETS <http://www.metamarkets.com/>m 765.490.0454|t @drcrallen
<https://twitter.com/drcrallen>charles.allen@metamarkets.com